Dockerfile

```diff
@@ -7,6 +7,6 @@ WORKDIR /app
 COPY . .
 RUN make build
 
-FROM registry.fedoraproject.org/fedora-minimal:38
+FROM cgr.dev/chainguard/glibc-dynamic:latest
```

Comment:
Are we confident that we can trust/rely on Chainguard's artifacts?

Reply:
From SIG meeting:

- the project has its source published
- @edeNFed is going to link the upstream Dockerfile so it can be reviewed
- @pellared can you post issues you have found with chainguard artifacts?

Reply:
> @edeNFed is going to link the upstream Dockerfile so it can be reviewed

The images are not created using Dockerfiles. Therefore the review is not easy. The images are creating using https://github.com/chainguard-dev/apko. The OS used by the images is https://github.com/wolfi-dev/ (also maintained by Chainguard)

> @pellared can you post issues you have found with chainguard artifacts?

Personally, I have not found any issues but I do not feel also competent enough 😢 


```
$ docker scout recommendations cgr.dev/chainguard/glibc-dynamic:latest
    ✓ Image stored for indexing
    ✓ Indexed 10 packages

Could not display recommendations for image  cgr.dev/chainguard/glibc-dynamic:latest 
image has no base image

$ docker scout cves cgr.dev/chainguard/glibc-dynamic:latest
    ✓ SBOM of image already cached, 10 packages indexed
    ✓ No vulnerable package detected


$ docker scout quickview cgr.dev/chainguard/glibc-dynamic:latest
    ✓ SBOM of image already cached, 10 packages indexed
ERROR   Status: image has no base image, Code: 1

$ trivy image cgr.dev/chainguard/glibc-dynamic:latest
2023-05-31T09:38:51.084+0200    INFO    Need to update DB
2023-05-31T09:38:51.084+0200    INFO    DB Repository: ghcr.io/aquasecurity/trivy-db
2023-05-31T09:38:51.084+0200    INFO    Downloading DB...
37.36 MiB / 37.36 MiB [------------------------------------------------------------------------------------------------------------------] 100.00% 8.19 MiB p/s 4.8s
2023-05-31T09:38:59.716+0200    INFO    Vulnerability scanning is enabled
2023-05-31T09:38:59.716+0200    INFO    Secret scanning is enabled
2023-05-31T09:38:59.716+0200    INFO    If your scanning is slow, please try '--scanners vuln' to disable secret scanning
2023-05-31T09:38:59.716+0200    INFO    Please see also https://aquasecurity.github.io/trivy/dev/docs/secret/scanning/#recommendation for faster secret detection
2023-05-31T09:38:59.977+0200    INFO    Detected OS: wolfi
2023-05-31T09:38:59.977+0200    INFO    Detecting Wolfi vulnerabilities...
2023-05-31T09:38:59.977+0200    INFO    Number of language-specific files: 0

cgr.dev/chainguard/glibc-dynamic:latest (wolfi 20230201)

Total: 0 (UNKNOWN: 0, LOW: 0, MEDIUM: 0, HIGH: 0, CRITICAL: 0)
```

I think that maybe also because of the way the images are created they are hard to be diagnosed by container image scanning tools.

The OS used by the images is https://github.com/wolfi-dev/ (also maintained by Chainguard).

Personally, I do not see any good reason to have confidence on using Chainguard's products. Especially in the light of:

![image](https://github.com/open-telemetry/opentelemetry-go-instrumentation/assets/5067549/cf0367f5-8dcc-4aeb-b061-6b346e8c4526)

_Source: https://www.chainguard.dev/chainguard-images_

How about trying to use Alpine and add `apk add gcompat`?